src/bcc_sensor.c

```diff
@@ -1757,3 +1757,66 @@ int trace_tcp_sendmsg(struct pt_regs *ctx, struct sock *sk, struct msghdr *msg)
 
 	return 0;
 }
+
+struct sched_process_exit_args {
+    __u64 pad;
+    char comm[16];
+    pid_t pid;
+    int prio;
+};
+
+int on_sched_process_exit(struct sched_process_exit_args *arg)
+{
+    struct data_t data = {};
+    struct task_struct *task;
+    unsigned int flags = 0;
+    pid_t tgid = 0;
+    pid_t pid = 0;
+    task = (struct task_struct *)bpf_get_current_task();
```

Comment:
The area of code with the task_struct usage should probably be wrapped in an ifdef since `bpf_get_current_task` doesn't exist in kernels older than 4.8.0.

As you can see, the ifdef on the tail end of this function
`#if LINUX_VERSION_CODE < KERNEL_VERSION(4, 8, 0)`

Reply:
ok good find. does the new commit look like it resolves this ok?